Pages/Login-Page/login.css

```diff
@@ -0,0 +1,55 @@
+body
+{
+    height: 100vh;
+}
+
+.login-content-container
+{
+    /* max-height: fit-content; */
+    height: 100%;
+    max-width: fit-content;
+    align-self: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+    color: var(--royal-blue);
+    padding: 5rem;
+    border-radius: 6px;
+    background-color: var();
```

Comment:
bg color missing, may be that was unintentional..can be removed if so.

Reply:
Oh right, I forgot to remove it. Thanks